File: ProyectoAerolinea/src/java/Model/Ciudad.java

package Model;

/**
 *
 * @author Greivin
 */
public class Ciudad implements Jsonable{
    
   public String codigo;
    public String nombre;

    public Ciudad(){
        this("","");
    }
    
    public Ciudad(String codigo, String nombre) {
        this.codigo = codigo;
        this.nombre = nombre;
    }

    public String getCodigo() {
        return codigo;
    }

    public void setCodigo(String codigo) {
        this.codigo = codigo;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
    
    
}

File: FE y BE/ProyectoAerolinea-FrontEnd/web/js/Ciudad.js
function Ciudad(codigo,nombre){
    this.Ciudad(codigo,nombre);
}

Ciudad.prototype={
    codigo:"",
    nombre:"",
    
    Ciudad: function(codigo,nombre){
        this.codigo=codigo;
        this.nombre=nombre;
    },
    
    toString: function(){
        return this.codigo;
    }
};

Ciudad.from= function(plain){
    ciudad = new Ciudad(plain.codigo,plain.nombre);
    return ciudad;
};

Ciudad.to = function(ciudad){
    return{
        
        _class:'Ciudad',
        codigo:ciudad.codigo,
        nombre:ciudad.nombre
    };
};